scripts/projectView.js

```diff
@@ -5,7 +5,6 @@ projectsView.handleMainNav = function() {
     e.preventDefault();
     $('.page-content').hide();
     var $clickEvent = $(this);
-    console.log($clickEvent);
     $('[id="' + $clickEvent.attr('data-content') + '"]').show();
     $('.top-menu').removeClass('expand');
   });
```

Comment:
i would take out the funciton inside your document.ready function. I would just call the function there.